deployments/kovan/usdc/roots.json

```diff
@@ -1,7 +0,0 @@
-{
-    "comet": "0xa7D85950E6E1bB7115c626cBB388Fa0a8C927c1c",
```

Comment:
did you mean to delete the Kovan roots?

Reply:
No good catch, just meant to do that to test Import logs